Formula/dosfstools.rb

```diff
@@ -0,0 +1,41 @@
+class Dosfstools < Formula
+  desc "Tools to create, check and label file systems of the FAT family"
+  homepage "https://github.com/dosfstools"
+
+  head "https://github.com/dosfstools/dosfstools.git"
+
+  stable do
+    url "https://github.com/dosfstools/dosfstools/releases/download/v4.0/dosfstools-4.0.tar.gz"
+    mirror "https://fossies.org/linux/misc/dosfstools-4.0.tar.gz"
+    sha256 "25809c30403c9a945ae34827ec75df32ff55017415a720864fefccc8c8f9991f"
+
+    patch do
+      # This patch restores the old defaults in versions up to v3.0.28
+      url "https://github.com/dosfstools/dosfstools/commit/1e76e5778a1885452939a79d9145b80634a5b023.patch"
+      sha256 "824c800a589db8a779fddbbd4b8eef5a2941f838ef8f46ad390a680c8cabd2a4"
+    end
+  end
+
+  option "with-symlinks", "Symlink older names of the tools to the current ones on installation"
```

Comment:
Can we change the option name to `with-compat-symlinks` (inspired by the corresponding `./configure` option)? The currently used `with-symlinks` feels very generic to me and isn't obvious without reading the option description.